applications/PfemSolidMechanicsApplication/custom_utilities/stress_invariants_utilities.hpp

```diff
@@ -8,7 +8,7 @@
 
 #if !defined(KRATOS_STRESS_INVARIANTS_UTILITIES)
 #define KRATOS_STRESS_INVARIANTS_UTILITIES
-#define PI 3.1415926535898
+#define m_PI 3.1415926535898
```

Comment:
```suggestion
```

better use [this](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/global_variables.h#L25)